### PR TITLE
Create icon component

### PR DESCRIPTION
**Pull request by CBinh710**
# Result
image

### CODE REVIEW

File: practice-1/src/index.html [hunk 1]
```diff
@@ -7,7 +7,12 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <div class="card">
+    <img src="assets/images/icons/cart.svg" class="icon" alt="cart-icon">
+    <img src="assets/images/icons/facebook.svg" class="icon-md" alt="facebook-icon">
```
Comment (lintran-agilityio) [inline practice-1/src/index.html]: when you create component and test the ui, can hold it in your local no need to push testing change in html

File: practice-1/src/styles/components/icon.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	border: none;
+}
+
+.icon-sm {
+	width: 20px;
+	height: 20px;
+}
+
+.icon-md {
+	width: 28px;
```
Comment (lintran-agilityio) [inline practice-1/src/styles/components/icon.css]: your mean circle icon why the width and height is 28px but border-radius is 62px?

File: practice-1/src/index.html (hunk 1 above)
Comment (CBinh710) [reply practice-1/src/index.html]: Thanks, i removed it

File: practice-1/src/styles/components/icon.css (hunk 2 above)
Comment (CBinh710) [reply practice-1/src/styles/components/icon.css]: I just set width and height to wrap icon, so border radius is unnecessary thanks for your contribution

**Pull request merged**